docs/source/manpage.rst

```diff
@@ -38,7 +38,7 @@ All options available as of Flake8 3.1.0::
     -q, --quiet           Report only file names, or nothing. This option is
                           repeatable.
     --count               Print total number of errors and warnings to standard
-                          error and set the exit code to 1 if total is not
+                          output and set the exit code to 1 if total is not
```

Comment:
this isn't meant to be kept up to date (see the text above it) so this can be discarded

in reality I should probably just remove this documentation page and suggest packagers to use help2man or whatever